README.md

```diff
@@ -70,16 +70,110 @@ To:
 }
 ```
 
-##Usage
+##How to use
 
+
+To use this library you first need to include it.
 ```javascript
 var dirTree = require('directory-tree');
-var tree = dirTree.directoryTree('/some/path');
 ```
 
-And you can also filter by extensions:
+After that you can turn every directory tree in a json object.
 
 ```javascript
 var dirTree = require('directory-tree');
-var filteredTree = dirTree.directoryTree('/some/path', ['.jpg', '.png']);
+
+// Get full directory tree
+var tree = dirTree.directoryTree('/photos');
+
+// Only get the structure which contains *.jpg or *.png files.
+var filteredTree = dirTree.directoryTree('/photos',  {'fileTypes': ['.jpg', '.png']} );
```

Comment:
This doesn't match the code (there's no `filetypes` in the code, there is `extensions`).